userguide/virtualmachines.rst

```diff
@@ -659,6 +659,27 @@ contain a space.
    Edit Rancher Password in Raw File Device
 
 
+SSH into the Docker Host
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Find the Docker Host :guilabel:`Com Port` on the
+:menuselection:`Virtual Machines`
+page. It has a value of :samp:`/dev/nmdm{1b}`, where :literal:`{1b}` is
```

Comment:
Com port names have the format :samp:...

(I think the variable part is uppercase, please verify.)